pkg/cmd/server/kubernetes/node.go

```diff
@@ -27,9 +28,12 @@ import (
 	utiliptables "k8s.io/kubernetes/pkg/util/iptables"
 	utilnet "k8s.io/kubernetes/pkg/util/net"
 	"k8s.io/kubernetes/pkg/util/sysctl"
+	"k8s.io/kubernetes/pkg/volume"
 
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	dockerutil "github.com/openshift/origin/pkg/cmd/util/docker"
+	"github.com/openshift/origin/pkg/volume/empty_dir"
```

Comment:
I think I actually picked that up from kube which has their plugin in an empty_dir package. Will change it for the quota wrapper.